cmd/Makefile.am

```diff
@@ -75,7 +75,14 @@ fmt-check:: $(wildcard $(addsuffix /*.[ch],$(addprefix $(srcdir)/,$(subdirs))))
 hack: snap-confine/snap-confine-debug snap-confine/snap-confine.apparmor snap-update-ns/snap-update-ns snap-seccomp/snap-seccomp snap-discard-ns/snap-discard-ns snap-device-helper/snap-device-helper snapd-apparmor/snapd-apparmor
 	sudo install -D -m 755 snap-confine/snap-confine-debug $(DESTDIR)$(libexecdir)/snap-confine
 	sudo setcap "$$(cat snap-confine/snap-confine.caps)" $(DESTDIR)$(libexecdir)/snap-confine
-	if [ -d $(DESTDIR)$(APPARMOR_SYSCONFIG) ]; then sudo install -m 644 snap-confine/snap-confine.apparmor $(DESTDIR)$(APPARMOR_SYSCONFIG)/$(patsubst .%,%,$(subst /,.,$(libexecdir))).snap-confine.real; fi
+	if [ -d $(DESTDIR)$(APPARMOR_SYSCONFIG) ]; then \
+		prof_name=$(patsubst .%,%,$(subst /,.,$(libexecdir))).snap-confine; \
```

Comment:
You can perhaps make PROF_NAME a goal-only variable and reference it as any other make variable?

```make
foo: VAR=value
   echo $(VAR)
```